Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/PaintBrushManipulator.cpp

```diff
@@ -10,74 +10,30 @@
 #include <AzFramework/Entity/EntityDebugDisplayBus.h>
 #include <AzToolsFramework/Manipulators/PaintBrushManipulator.h>
 #include <AzToolsFramework/Manipulators/PaintBrushNotificationBus.h>
-#include <AzToolsFramework/Manipulators/PaintBrushRequestBus.h>
 #include <AzToolsFramework/Manipulators/ManipulatorSnapping.h>
 #include <AzToolsFramework/Manipulators/ManipulatorView.h>
+#include <AzToolsFramework/PaintBrushSettings/PaintBrushSettingsNotificationBus.h>
+#include <AzToolsFramework/PaintBrushSettings/PaintBrushSettingsRequestBus.h>
+#include <AzToolsFramework/PaintBrushSettings/PaintBrushSettingsWindow.h>
 #include <AzToolsFramework/Viewport/ViewportMessages.h>
 #include <AzToolsFramework/ViewportSelection/EditorSelectionUtil.h>
 
 namespace AzToolsFramework
 {
-    void PaintBrushConfig::Reflect(AZ::ReflectContext* context)
+    namespace
     {
-        if (auto serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
-        {
-            serializeContext->Class<PaintBrushConfig>()
-                ->Version(1)
-                ->Field("Radius", &PaintBrushConfig::m_radius)
-                ->Field("Intensity", &PaintBrushConfig::m_intensity)
-                ->Field("Opacity", &PaintBrushConfig::m_opacity);
+        // Increase / decrease paintbrush radius amount in meters.
+        static constexpr float PaintbrushRadiusAdjustAmount = 0.25f;
```

Comment:
suggestion: We might want to move this to an AZ_CVAR (or settings registry) so UX can easily tweak this without recompiling